stdlib/list.gr

```diff
@@ -1144,6 +1176,40 @@ provide module Associative {
     }
   }
 
+  let rec setAllHelp = (key, value, list, hitMatch=false) => {
+    match (list) {
+      [] when !hitMatch => [(key, value)],
+      [] => [],
+      [(k, _), ...rest] when key == k =>
+        [(k, value), ...setAllHelp(key, value, rest, hitMatch=true)],
+      [first, ...rest] =>
+        [first, ...setAllHelp(key, value, rest, hitMatch=hitMatch)],
+    }
+  }
+  /**
+   * Creates a new list with all values in the list of key-value pairs that match the key replaced.
```

Comment:
Not really sure how I feel about the `setAll` function tbh, not sure when you would want this

Reply:
Me neither but I wanted the api's to match and I didn't see any harm in having it.